### ISSUE TITLE
[Bug Report]: Cannot scaffold an app with vite using create-vite

### ISSUE THREAD

**Issue by Tuditi** [tauri-apps/create-tauri-app#103]
I'm not able to scaffold a Svelte-TS app with vite using the `create-tauri cli app`.

Steps to reproduce:

1) Run `yarn create tauri-app`
2) Answer configuration questions:
```
? What is your app name? tauri-app
? What should the window title be? Tauri App
? What UI recipe would you like to add? **create-vite** (vanilla, vue, react, 
svelte, preact, lit) 
(https://vitejs.dev/guide/#scaffolding-your-first-vite-project)
? Add "@tauri-apps/api" npm package? Yes
? Which vite template would you like to use? **svelte-ts**
```
3) cd tauri-app
4) yarn tauri dev
5) Tauri window opens, but throws the following error: `SyntaxError: Importing binding name 'default' cannot be resolved by star export entries.`

Software versions:
- yarn: 1.22.19
- @tauri-apps/cli: 1.0.5
- create-tauri-app: 1.0.2

**Comment by amrbashir** [comment]
Can you try again but when prompted to install `@tauri-apps/api`, answer No

**Comment by Tuditi** [comment]
I have the same error

**Comment by amrbashir** [comment]
it works fine for me, what OS are you using? try `yarn create vite` and choose `svelte-ts` and see if the error happens

**Comment by Tuditi** [comment]
I'm using Ubuntu 20.04.4 LTS.

`yarn create vite` seems to work, but that uses `vite` when running `yarn dev`, which also works in `yarn create tauri-app`. The problem arises when running `yarn tauri dev`.

It seems like others run into similar issues: https://discord.com/channels/616186924390023171/625037620996734986/1005055153742823424

**Comment by codeaye** [comment]
Uhm quick addition, im on mac and i tried with ts-svelte and react-ts on vite using the create-tauri-app.
It seems to not load the content in dev mode but does so perfectly in build mode

**Comment by codeaye** [comment]
<img width="178" alt="image" src="https://user-images.githubusercontent.com/62173713/183097369-73a6efab-6e6d-44d5-abac-9feb7ef25fb7.png">
<img width="399" alt="image" src="https://user-images.githubusercontent.com/62173713/183097409-9ef32268-313a-4c8d-a06a-0debe2f847db.png">
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/62173713/183097454-15795ff5-aa77-46d6-94a2-bb73aca2c9dc.png">

! Done worry about no decorators, i disabled that specificly

**Comment by codeaye** [comment]
Local host works just fine soo its a tauri issue...

**Comment by codeaye** [comment]
@amrbashir i think i found the issue... so the local host for vite for me is `http://localhost:5174/` when it used to be `http://localhost:5173/` but create tauri app still configures tauri.conf.json to use the old one... might wanna change that

**Comment by codeaye** [comment]
https://media.discordapp.net/attachments/1005055153742823424/1005059325372399746/unknown.png?width=1820&height=1138

while the same thing done in svelte-ts has vite configured to localhost:5175

**Comment by FabianLars** [comment]
5173 is actually the correct port. But if it's taken already vite just adds `1` that's why you got 5174 and 5175. Most likely orphaned processes :/

**Comment by codeaye** [comment]
yes i found that dont know why its like that on my system as far as im aware thats the only running one....

**Comment by FabianLars** [comment]
Yeah the prior tauri/vite dev runs weren't able to close the devserver correctly, but still detached it from the console, so you'd need to look into the process manager or whatever it's called in mac, and close the nodejs processes (or restart the system) :/

**Comment by codeaye** [comment]
ye im trynna find that process...

**Comment by codeaye** [comment]
`sudo lsof -i :5173` found it and i killed it, it works now, but really sure where the process was running from...

**Comment by amrbashir** [comment]
Closing this issue as it should be fixed in the rewrite in #90
